Signal/src/ViewControllers/HomeView/HomeViewController.m

```diff
@@ -913,6 +913,13 @@ - (void)updateSearchResultsVisibility
     [self.searchResultsController updateSearchResultsWithSearchText:searchText];
     BOOL isSearching = searchText.length > 0;
     self.searchResultsController.view.hidden = !isSearching;
+
+    if (isSearching) {
```

Comment:
We show the search results table view "on top of" the normal home view table view.  Don't let the user scroll the "background" table view (by swiping on the search field) while the "foreground" table view is visible.